bin/ch/ch.vcxproj

```diff
@@ -37,6 +37,7 @@
         ole32.lib;
         kernel32.lib;
         Rpcrt4.lib;
+        $(ChakraCommonLinkDependencies);
```

Comment:
Do we really need to add this link dependency? Is there a more lightweight way to get the Dll name?